dirty.html

```diff
@@ -1,186 +1,185 @@
 <html>  
   <head>
-    <h1>WDI London September 2013
+    <h1>WDI London September 2013</h1>
     <meta value="description" content="Web development immersive ">
   </head>
   <div class="body">
-    <div>
-        <div>
-        </div>
+    <div> <!-- deleted extra <div></div>-->
         <div>
-          <div>Web Development Immersive <div>General&nbsp;Assembly:&nbsp;Autumn/Winter&nbsp;2013</div></div>
+          <div>Web Development Immersive </div>General&nbsp;Assembly:&nbsp;Autumn/Winter&nbsp;2013<div></div><!-- corrected </div> after "Immersive"-->
```

Comment:
If you look at this line, things you should notice:
- `div` is a block element, if I have `div`s within `div`s, you should express that in the code by indenting them in separate lines,
- In this case, you don't need separate `div`s, but rather some inline styling element if anything, like `span`
- Remove the `&nbsp`s, these are seldom needed. i.e. when you want to have many spaces in a row " &nbsp; nbsp; "